lib/adapter.js

```diff
@@ -161,16 +164,17 @@ class MochaAdapter {
         return message
     }
 
-    requireExternalModules (compilers = [], requires = []) {
-        compilers.concat(requires).forEach((mod) => {
-            mod = mod.split(':')
-            mod = mod[mod.length - 1]
+    requireExternalModules (modules, context) {
+        modules.forEach(module => {
```

Comment:
since you can set multiple requires in mocha this should at least support arrays